tests/testthat/integration-server.R

```diff
@@ -1,6 +1,8 @@
-server <- porcelain::porcelain_background$new(
-  api, args = list(queue_id = paste0("hintr:", ids::random_id())))
-server$start()
+withr::with_dir(testthat::test_path(), {
```

Comment:
we should add something to porcelain_bg to make this nicer

Reply:
Made a ticket https://mrc-ide.myjetbrains.com/youtrack/issue/RESIDE-333/Add-ability-to-start-porcelain-server-with-specified-working-directory